tests/cypress/cypress.json

```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "https://d2dr7dgo9g0124.cloudfront.net",
+  "baseUrl": "https://d2e4impk96qz2a.cloudfront.net",
```

Comment:
Probably want to change this back? or is it only used for local testing?

Reply:
It's local but I should change it back!